addons/hr_expense/hr_expense_demo.xml

```diff
@@ -35,61 +35,60 @@
             <field name="name">Hotel Accommodation</field>
             <field name="default_code">HA0</field>
             <field name="categ_id" ref="hr_expense.cat_expense"/>
-            <field name="hr_expense_ok" eval="True" />
+            <field name="can_be_expensed" eval="True"/>
             <field name="image" type="base64" file="hr_expense/static/img/hotel_rent-image.jpg"/>
         </record>
-        <record id="expenses0" model="hr.expense.expense">
-            <field name="name">Expenses</field>
+
+        <record id="expenses0" model="hr_expense.sheet">
+            <field name="name">Expenses for Administrator</field>
+            <field name="employee_id" ref="hr.employee"/>
             <field name="date" eval="time.strftime('%Y-08-25')"/>
+            <field name="employee_payable_account_id" ref="account.a_pay"/>
         </record>
```

Comment:
No need to create an expense sheet. It will be automatically created when you submit the expense line !!!